src/bandersnatch/tests/test_mirror.py

```diff
@@ -92,13 +95,52 @@ def test_mirror_generation_4_resets_status_files(tmpdir):
     assert open(str(tmpdir/'generation'), 'r').read() == '5'
 
 
-def test_mirror_remove_blacklisted_packages(tmpdir):
-    blacklisted_package = 'example1'
-    m = Mirror(str(tmpdir), mock.Mock(),
-               package_blacklist=[blacklisted_package])
-    m.packages_to_sync = {'example1': None, 'example2': None}
-    m._remove_blacklisted_packages()
-    assert blacklisted_package not in m.packages_to_sync
+def test_mirror__filter_packages__match(tmpdir):
```

Comment:
Why double __ ? What does it mean?

Reply:
It is to differentiate between spaces and underscores since the function names being tested have underscores in their name and I like the test to include the name of the function being tested.

Reply:
Is this a standard or something? Never heard of it.